services/apps/src/apps/service/workspace.clj

```diff
@@ -1,9 +1,11 @@
 (ns apps.service.workspace
   (:use [korma.db :only [transaction]])
-  (:require [apps.persistence.workspace :as wp]))
+  (:require [apps.clients.iplant-groups :as iplant-groups]
+            [apps.persistence.workspace :as wp]))
 
 (defn get-workspace
-  [{:keys [username]}]
+  [{short-username :shortUsername :keys [username]}]
+  (iplant-groups/add-de-user short-username)
```

Comment:
Would this make more sense to have in the `bootstrap` endpoint rather than in the `get-workspace` function?
It makes sense to get-or-create the workspace in this function, but I wouldn't think to look here when trying to figure out where users are added to `iplant-groups`.